share/normalize/links.py

```diff
@@ -284,3 +289,14 @@ def __init__(self, string):
 
     def execute(self, obj):
         return self._string + obj
+
+
+class XPathLink(AbstractLink):
+    def __init__(self, xpath):
+        self._xpath = xpath
+        super().__init__()
+
+    def execute(self, obj):
+        obj = xmltodict.unparse(obj)
+        obj = etree.XML(obj.encode())
+        return obj.xpath(self._xpath)
```

Comment:
Return the re-parsed version here `xmltodict.parse(etree.tostring(obj.xpath(self._xpath)))`

That way if someone decided to use `xpath